src/_common/content/components/media-item/media-item.ts

```diff
@@ -164,8 +168,9 @@ export default class AppContentMediaItem extends Vue {
 		return !this.mediaItem?.is_animated && !!this.mediaItem?.mediaserver_url;
 	}
 
-	created() {
-		this.mediaItem = createMediaItemFromContentOwner(this.owner, this.mediaItemId);
+	async created() {
```

Comment:
shouldn't make these hooks async, it also won't properly assign in SSR I think? does this make it so taht the media doesn't show anymore in SSR?

Reply:
This was breaking SSR - change to a different usage of creating the media item.